devel/libical/Portfile

```diff
@@ -35,22 +32,33 @@ subport libical-devel {
 }
 
 if {${subport} eq "${name}"} {
-    # v 1.0.1
-    checksums       rmd160  780f9fb08b324536af362f8812ff671e4d1082d0 \
-                    sha256  b3898c9ca3522ba4f0454e8b7c26a85b2b1259e16d670cccf7abb035c8a6caa9
+    # v 2.0.0
+    checksums       rmd160  1901573af1fb57be8feca3b3dd9db758c3e136ee \
+                    sha256  c9c076153867cd6ebac23febb276036597443eabd30ba4534d38950a2cbb889e
 } else {
-    # v1.0.1-351-g25bc500
-	checksums       rmd160  4379c5d6d31a78ecb2ceaff6e64a8900e9b30cfe \
-	                sha256  f7c22344a461f8ee1ce65e3efa477e9846b1ca39f44af622a149ce37f3850332
+    # v2.0.0-115-g1b83385
```

Comment:
A matter of personal preference and completely orthogonal to your changes. I would hate to write the version at several different places as I would probably forget to change it next time. You could also put the checksums right below the version definition.